src/main/java/model/Figure.java

```diff
@@ -0,0 +1,35 @@
+package model;
+
+public abstract class Figure {
+    private String color;
+    private String figureName;
+
+    public Figure(String color) {
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColour(String colour) {
+        this.color = colour;
+    }
+
+    public String getFigureName() {
+        return figureName;
+    }
+
+    public void setFigureName(String figuretype) {
+        this.figureName = figuretype;
+    }
+
+    public void drow() {
+        System.out.println("Info about figure");
+    }
+
+    public abstract double getArea();
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-use-abstract-classes-to-set-behaviour-for-classes)